pkg/cmd/fixcerts/fixcerts.go

```diff
@@ -0,0 +1,431 @@
+		return fmt.Errorf("failed to wait for recovery apiserver to be ready: %v", err)
+	}
+	klog.Info("Recovery apiserver is up")
+
+	// Run cert recovery
+
+	// We already have kubeClient created
+	kubeClient, err := apiserver.GetKubeClientset()
+	if err != nil {
+		return fmt.Errorf("failed to get kubernetes clientset: %v", err)
+	}
+
+	restConfig, err := apiserver.RestConfig()
+	if err != nil {
+		return fmt.Errorf("failed to get kubernetes rest config: %v", err)
+	}
+
+	operatorConfigClient, err := operatorversionedclient.NewForConfig(restConfig)
+	if err != nil {
+		return err
+	}
+
+	configClient, err := configeversionedclient.NewForConfig(restConfig)
+	if err != nil {
+		return fmt.Errorf("failed to create config client: %v", err)
+	}
+
+	certRotationCtx, certRotationCtxCancel := context.WithCancel(ctx)
+	defer certRotationCtxCancel()
+
+	certRotationWg := sync.WaitGroup{}
+
+	operatorConfigInformers := operatorexternalinformers.NewSharedInformerFactory(operatorConfigClient, 10*time.Minute)
+	certRotationWg.Add(1)
+	go func() {
+		defer certRotationWg.Done()
+		operatorConfigInformers.Start(certRotationCtx.Done())
+	}()
+
+	configInformers := configexternalinformers.NewSharedInformerFactory(configClient, 10*time.Minute)
+	certRotationWg.Add(1)
+	go func() {
+		defer certRotationWg.Done()
+		configInformers.Start(certRotationCtx.Done())
+	}()
+
+	kubeApiserverOperatorClient := &kubeapiserveroperatorclient.OperatorClient{
+		Informers: operatorConfigInformers,
+		Client:    operatorConfigClient.OperatorV1(),
+	}
+
+	kubeControllerManagerOperatorClient := &kubecontrollermanageroperatorclient.OperatorClient{
+		Informers: operatorConfigInformers,
+		Client:    operatorConfigClient.OperatorV1(),
+	}
+
+	kubeApiserverInformersForNamespaces := v1helpers.NewKubeInformersForNamespaces(
+		kubeClient,
+		"",
+		kubeapiserveroperatorclient.GlobalUserSpecifiedConfigNamespace,
+		kubeapiserveroperatorclient.GlobalMachineSpecifiedConfigNamespace,
+		kubeapiserveroperatorclient.TargetNamespace,
+		kubeapiserveroperatorclient.OperatorNamespace,
+		"kube-system",
+	)
+	certRotationWg.Add(1)
+	go func() {
+		defer certRotationWg.Done()
+		kubeApiserverInformersForNamespaces.Start(certRotationCtx.Done())
+	}()
+
+	kubeControllerManagerInformersForNamespaces := v1helpers.NewKubeInformersForNamespaces(
+		kubeClient,
+		"",
+		kubecontrollermanageroperatorclient.GlobalUserSpecifiedConfigNamespace,
+		kubecontrollermanageroperatorclient.GlobalMachineSpecifiedConfigNamespace,
+		kubecontrollermanageroperatorclient.TargetNamespace,
+		kubecontrollermanageroperatorclient.OperatorNamespace,
+		"kube-system",
+	)
+	certRotationWg.Add(1)
+	go func() {
+		defer certRotationWg.Done()
+		kubeControllerManagerInformersForNamespaces.Start(certRotationCtx.Done())
+	}()
+
+	eventRecorder := events.NewKubeRecorder(kubeClient.CoreV1().Events(""), "fix-certs (CLI)", &corev1.ObjectReference{
+		APIVersion: "v1",
+		Kind:       "namespace",
+		Name:       "kube-system",
```

Comment:
use the `openshift-kube-apiserver-operator` namespace and set the namespace value or we end up in default and unfindable.